build.gradle.kts

```diff
@@ -266,6 +270,7 @@ project(":idea") {
 project(":clion") {
     intellij {
         version = clionVersion
+        alternativeIdePath = clionLocalPath
```

Comment:
It affects nothing because this property makes sense only for root module

Reply:
Ah, I didn't know this. I thought that was for the `runIde` action.